Makefile

```diff
@@ -33,6 +33,9 @@ docker:
 	docker build . \
 		--build-arg ADD_BUILD_TAGS=$(ADD_BUILD_TAGS) \
 		--label "git_sha=$(GIT_SHA)" \
+		--build-arg http_proxy=$(HTTP_PROXY) \
+		--build-arg https_proxy=$(HTTPS_PROXY) \
+		--build-arg no_proxy=$(NO_PROXY) \
```

Comment:
Should these use the lowercase form as well? aka `--build-arg http_proxy=$(http_proxy)`, which i think can just be shortened to `--build-arg http_proxy`

Reply:
I followed what was present in the device-onvif-service, that way the environment variables only need to be set with one format for building both services...

Reply:
ok, that seems fair, as long as the documentation matches up, which I believe it does when I reviewed that last week.